runtime/onert/backend/cpu/ops/ConvolutionLayer.cc

```diff
@@ -197,6 +227,15 @@ void ConvolutionLayer::prepare()
     kernel.prepareQuant(getShape(_input), getShape(_kernel), getShape(_output), _strideWidth,
                         _strideHeight, _dilationWidthFactor, _dilationHeightFactor);
   }
+  else if (_input->data_type() == OperandType::QUANT_INT8_ASYMM && _kernel->is_constant() &&
+           !_input->is_dynamic() && !_output->is_dynamic())
+  {
+    kernel.getQuantizedConvolutionMultipliersAndShifts(
```

Comment:
This function will be called when input, kernel, and output tensors are constant. You need to call `getQuantizedConvolutionMultipliersAndShifts` at inference time to get proper quantized parameters for dynamic tensors.